Apps/Admin/Client/Pages/Support.razor.cs

```diff
@@ -79,6 +83,15 @@ protected override void OnInitialized()
         {
             base.OnInitialized();
             this.ResetState();
+
+            Uri uri = this.NavigationManager.ToAbsoluteUri(this.NavigationManager.Uri);
+
+            if (QueryHelpers.ParseQuery(uri.Query).TryGetValue(UserQueryType.HDID.ToString(), out var hdid))
```

Comment:
Nit: I think our style rules might complain about using `var` instead of `string` here.

Reply:
I changed out to StringValues.